config/application.rb

```diff
@@ -28,6 +25,9 @@ class Application < Rails::Application
     config.plugins = [:dynamic_form]
 
     config.autoload_paths << Rails.root.join('lib')
+
+    # Do not halt callback chains when a callback returns false.
+    ActiveSupport.halt_callback_chains_on_return_false = false
   end
```

Comment:
I was hoping I would change config after we are sure that changes up to this point don't break anything.
However, now it seems like it is going to be a while before this gets deployed.. I have added the changes of `rails app:update`. Thanks for having a look at the PR :)